apicapi/apic_client.py

```diff
@@ -515,10 +566,51 @@ def login(self, usr=None, pwd=None):
                                          err_text=attributes['text'],
                                          err_code=attributes['code'])
 
+    def set_private_key(self, usr, cert_name, private_key_file):
```

Comment:
This method is equivalent to the login method - login for password authentication, this method for certificate authentication.